packages/components/cascader/src/cascader.ts

```diff
@@ -55,6 +61,10 @@ export const cascaderProps = buildProps({
     type: Boolean,
     default: true,
   },
+  suffixIcon: {
+    type: iconPropType,
+    default: ArrowDown,
```

Comment:
**Suggestion:** since we have already made a fallback in the code, I think we can just leave this as `undefined`.